### PR TITLE
E2e fixtures navigate transactions

### PR DESCRIPTION
**Pull request by PeterYinusa**
Related: #11136

Explanation:  
  - create navigate transactions state with 4 unapproved transactions
  - moved navigate transactions test
  - updated navigate transactions test to use fixtures

### CODE REVIEW

**Review by brad-decker — APPROVED** [review]
😍 LGTM 🎉

File: test/e2e/tests/navigate-transactions.spec.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'navigate-transactions',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
```
Comment (darkwing) [inline test/e2e/tests/navigate-transactions.spec.js:24]: Nit: I've noticed in our E2E's that we reuse a values like this password, the secret key, etc.  Should we, maybe in a different PR, create a `constants` file and use vars from there?

**Review by darkwing — APPROVED** [review]

File: test/e2e/tests/navigate-transactions.spec.js (hunk 1 above)
Comment (brad-decker) [reply test/e2e/tests/navigate-transactions.spec.js:24]: that's a fantastic idea.

Comment (PeterYinusa) [reply test/e2e/tests/navigate-transactions.spec.js:24]: 💯 - i'll revisit that as a separate piece of work

**Pull request merged**